packages/base/src/state/puppeteer_native.rs

```diff
@@ -0,0 +1,202 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::Addr;
+use cosmwasm_std::Decimal256;
+use cosmwasm_std::Uint128;
+use cw_storage_plus::Item;
+
+use drop_puppeteer_base::state::BalancesAndDelegationsState;
+use drop_puppeteer_base::state::Transfer;
+
+use crate::msg::puppeteer::MultiBalances;
+
+use super::puppeteer::DropDelegation;
+
+#[cw_serde]
+pub struct ConfigOptional {
+    pub remote_denom: Option<String>,
+    pub allowed_senders: Option<Vec<String>>,
+    pub native_bond_provider: Option<Addr>,
+}
+
+#[cw_serde]
+pub struct Config {
+    pub remote_denom: String,
+    pub allowed_senders: Vec<Addr>,
+    pub native_bond_provider: Addr,
+}
+
+pub const CONFIG: Item<Config> = Item::new("config");
+
+pub const NON_NATIVE_REWARD_BALANCES: Item<BalancesAndDelegationsState<MultiBalances>> =
+    Item::new("non_native_reward_balances");
+
+pub const RECIPIENT_TRANSFERS: Item<Vec<Transfer>> = Item::new("recipient_transfers");
```

Comment:
This item is never used anywhere and is never written to. I suggest deleting it.

Note: However, it is being read in `fn query_transactions()`. If you want to save `QueryMsg::Transactions {}` for backwards compatibility, it is possible to do so with this trick in `contract.rs`:

```rust
fn query_transactions() -> ContractResult<Binary> {
    Ok(to_json_binary::<[(); 0]>(&[])?)
}
```

This way this query will always return an empty array.